From 17f2f33d8733aa7dcb5e52fc9043447758a74869 Mon Sep 17 00:00:00 2001
From: orbea <orbea@riseup.net>
Date: Fri, 4 Sep 2020 09:21:45 -0700
Subject: [PATCH] cmake: Fix compatibility with cmake 3.5.2.

cmake 3.5.2 does not have VERSION_GREATER_EQUAL or VERSION_LESS_EQUAL.
---
 cmake/modules/FindGLM.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/modules/FindGLM.cmake b/cmake/modules/FindGLM.cmake
index 32ce5058d..e3905b6c3 100644
--- a/cmake/modules/FindGLM.cmake
+++ b/cmake/modules/FindGLM.cmake
@@ -52,6 +52,6 @@ find_package_handle_standard_args(GLM
 )
 
 # Enable GLM experimental extensions for GLM 0.9.9.0 to 0.9.9.3
-if(GLM_VERSION_STRING VERSION_GREATER_EQUAL 0.9.9.0 AND GLM_VERSION_STRING VERSION_LESS_EQUAL 0.9.9.3)
+if((GLM_VERSION_STRING VERSION_EQUAL 0.9.9.0 OR GLM_VERSION_STRING VERSION_GREATER 0.9.9.0) AND (GLM_VERSION_STRING VERSION_EQUAL 0.9.9.3 OR GLM_VERSION_STRING VERSION_LESS 0.9.9.3))
   add_definitions(-DGLM_ENABLE_EXPERIMENTAL)
 endif()
-- 
2.25.0

